crypto/cmp/cmp_asn.c

```diff
@@ -287,23 +287,31 @@ OSSL_CMP_ITAV *OSSL_CMP_ITAV_new_rootCaKeyUpdate(const X509 *newWithNew,
                                                  const X509 *oldWithNew)
 {
     OSSL_CMP_ITAV *itav;
-    OSSL_CMP_ROOTCAKEYUPDATE *upd = OSSL_CMP_ROOTCAKEYUPDATE_new();
+    OSSL_CMP_ROOTCAKEYUPDATE *upd = NULL;
 
+    if (newWithNew == NULL)
+        goto null_value;
+
+    upd = OSSL_CMP_ROOTCAKEYUPDATE_new();
```

Comment:
`doc/man3/OSSL_CMP_ITAV_new_caCerts.pod` needs to be updated to cover the new use case for `OSSL_CMP_ITAV_get0_rootCaKeyUpdate()`.